src/core/decoder.js

```diff
@@ -38,14 +38,8 @@ function decodePoint(vertices) {
     };
 }
 
-function clip(x) {
-    if (x > 1) {
-        return 1;
-    }
-    if (x < -1) {
-        return -1;
-    }
-    return x;
+function isClipped(l) {
+    return l[0] == -1 || l[0] == 1 || l[1] == -1 || l[1] == 1;
```

Comment:
Could we use `abs` here?

Reply:
How?

Reply:
`Math.abs(l[0]) == 1 ||  Math.abs(l[1]) == 1` ?

Reply:
Why is that better?

Reply:
Not very huge improvement though, just a suggestion to improve readability. But it is fine that way :+1: